ewidth.py

```diff
@@ -1,182 +1,221 @@
-
+ 
 # Jean-paul Ventura
 # January 12, 2017
 # Modified: February 20th, 2016
 
 from astropy.io import fits
 import numpy as np
 
-# Calculate equivalent width
+
 def eqwidth(filename,wavelength1,wavelength2,wavelength3,wavelength4):
-"""
-This function measures the equivalent width of spectral emission/absorption features.
+	"""
 
-====================
-Function parameters: 
-====================
+	This function measures the equivalent width of spectral emission/absorption features.
+	
+	====================
+	Function parameters: 
+	====================
+	
+	filename 	- SDSS spectra .fits filename entered as a string. You may specify entire
+				  path as a string, or if file is located in the current working directory, 
+				  simply the filename (e.g. 'SDSS-12345.fits') 
+	
+	wavelength1 - wavelength value where continuum region to the left of the spectral feature begins.
+	
+	wavelength2 - wavelength value where continuum region to the left of the spectral feature ends.
+	
+	wavelength3 - wavelength value where continuum region to the right of the spectral feature begins.
+	
+	wavelength4 - wavelength value where continuum region to the right of the spectral feature ends.
+	
+	=================
+	Function returns:
+	=================
+	
+	 eqwi - The measured equivalent width of the spectral emission/absorption feature
 
-filename 	- SDSS spectra .fits filename entered as a string. You may specify entire
-			  path as a string, or if file is located in the current working directory, 
-			  simply the filename (e.g. 'SDSS-12345.fits') 
+	"""
 
-wavelength1 - wavelength value where continuum region to the left of the spectral feature begins.
+	# Open .fits file 
+	hdu = fits.open(filename) 
 
-wavelength2 - wavelength value where continuum region to the left of the spectral feature ends.
+	# Define wavelength and flux arrays by accessing .fits sloan data
```

Comment:
Does this code only work for sloan data? If so, include that in the description of the function in your docstring/readme